client/layout/store-alerts/index.js

```diff
@@ -127,12 +129,20 @@ class StoreAlerts extends Component {
 				/>
 				{ actions && <div className="woocommerce-store-alerts__actions">{ actions }</div> }
 			</Card>
+		) : (
+			<StoreAlertsPlaceholder hasMultipleAlerts={ wcSettings.alertCount > 1 } />
 		);
 	}
 }
 
 export default compose(
 	withSelect( select => {
+		const { alertCount } = wcSettings;
+
+		if ( ! alertCount || 0 === alertCount ) {
```

Comment:
I think we should eliminate this check - if there are no notes, it will never check for new notes. The current behavior respects the `freshness` interval and periodically checks for new notes.

Reply:
That makes sense, reverted that commit.